tests/tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+current_directory=$(pwd)
+test_framework_directory="$current_directory/.jenkinsfile-runner-test-framework"
+
+. $test_framework_directory/init-jfr-test-framework.inc
+
+oneTimeSetUp() {
+  echo "Compiling CWP"
+  mvn clean package -f "$test_framework_directory/source/pom.xml"
```

Comment:
It runs unit tests here, but not integration tests for CWP. Not sure whether you want to have tests executed here at all

Reply:
Actually we want the opposite (not to execute them) since this Maven execution happens inside the integration tests execution.